torch/csrc/jit/ir.cpp

```diff
@@ -218,7 +218,8 @@ static std::ostream& indent(std::ostream& out, size_t level) {
 std::ostream& Node::print(
     std::ostream& out,
     size_t level,
-    std::vector<const Node*>* groups) const {
+    std::vector<const Node*>* groups,
+    bool debug) const {
```

Comment:
can we use a more descriptive name than debug? Like, printSourceLocations?

Reply:
I wanted to leave this general in case we wanted to add more info in the future but sure

Reply:
I feel like this version should be the default and we have something else that omits it if we want to parse it.